workflow/artifacts/oss/oss.go

```diff
@@ -45,7 +47,18 @@ func (ossDriver *ArtifactDriver) Load(inputArtifact *wfv1.Artifact, path string)
 				return false, err
 			}
 			objectName := inputArtifact.OSS.Key
-			err = bucket.GetObjectToFile(objectName, path)
+			origErr := bucket.GetObjectToFile(objectName, path)
+			if origErr == nil {
+				return true, nil
+			}
+			if origErr.(oss.ServiceError).Code != "NoSuchKey" {
```

Comment:
should probably check the cast is valid - to prevent nil panic

Reply:
Good idea. Updated.